Directory.Build.targets

```diff
@@ -19,7 +19,7 @@
   <ItemGroup Condition="'$(UnitTestType)' == 'xunit'">
     <PackageReference Include="xunit" Version="$(XUnitVersion)" />
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XUnitRunnerVersion)" />
-    <PackageReference Include="NunitXml.TestLogger" Version="$(NunitXmlTestLoggerVersion)" />
+    <PackageReference Include="XunitXml.TestLogger" Version="$(XunitXmlTestLoggerVersion)" />
```

Comment:
Note I am shamelessly stealing this bit for [this PR](https://github.com/dotnet/fsharp/pull/17782/files) :)

Reply:
Please, do!

Reply:
Okay so that one is merged, I resolved conflicts here :)